packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,140 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[
+            $style['c-mediaElement'],
+            (stacked ? $style['c-mediaElement--stack'] : ''),
+            (reverse ? $style['c-mediaElement--reverse'] : '')
+        ]">
+        <div
+            :class="[
+                $style['c-mediaElement-content'],
+                contentAlignClass
+            ]">
+            <h3
+                :class="[
+                    $style['c-mediaElement-title'],
+                    fontSizeClass
+                ]">
+                {{ title }}
+            </h3>
+            <p
+                :class="[
+                    $style['c-mediaElement-text'],
+                    fontSizeClass
+                ]">
+                {{ text }}
+            </p>
+        </div>
+        <div
+            :class="[
+                $style['c-mediaElement-imgWrapper'],
+                imageAlignClass
+            ]">
+            <img
+                :class="[$style['c-mediaElement-img'], 'c-mediaElement-img--override']"
+                :src="imageUrl"
+                alt="Media Element Image">
```

Comment:
Doesn't need to be part of this PR, but would be good to be able to pass in an `alt` for this image.

Reply:
Have added a prop for this